tests/files/client-test-pod-userpvc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: smbclient-withpvc
```

Comment:
Is this used for anything now?

Reply:
You are right. This is no longer used and just something I missed when updating the code.